File: week1_basic13/7.py
# Replace any negative array values with 'Dojo'.

def dojo(arr):
    for i in range(0, len(arr)-1, 1):
        if arr[i] < 0:
            arr[i] = "dojo"
            
    print(arr)
    

dojo([1,3,-4,5,-7,9,-11,12])

File: interview_prep/week1_basic13/7.js
// Replace any negative array values with 'Dojo'.


function dojo(arr) {
    for (let i=0; i < arr.length; i++) {
        if (arr[i] < 0) {
            arr[i] = "dojo"
        }
    }console.log(arr)
    
}

dojo([1,3,-4,5,-7,9,-11,12])



const dojoEs6 = arrEs6 => {
    const result = arrEs6.map(num => num < 0 ? "dojo" : num)

    console.log(result)
}

dojoEs6([1,3,-4,5,-7,9,-11,12])

